obal/data/roles/diff_package/tasks/koji_tag.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Find local NEVR for package
+  nevr:
+    spec_file: "{{ spec_file_path }}"
+    scl: "{{ tag.scl | default(None) }}"
+    dist: "{{ tag.dist | default(None) }}"
```

Comment:
shouldn't these be `default(omit)`? `None` is so pythonic, but this is YAML/Ansible :)